book/index.md

```diff
@@ -39,6 +39,55 @@ API resources in the second row (PipelineRun, Snapshot) should primarilly be tho
 data-plane resources. The system responds to user requests by creating and managing the lifecycle of
 these resources.
 
+- An [Application] represents a functionally coherent set of git branches that should be built,
+  tested, and released together. The user provides and names their [Applications]. They are
+  generally long-lived and don't change much after they are created.
```

Comment:
Nit: Not changing much after they are created seems like a misclassification. They are not changing much after they have been fully defined, but it may take some time for them to be fully defined (i.e. adding components individually).